tests/integration/components/power-calendar-multiple-test.js

```diff
@@ -48,20 +48,20 @@ module('Integration | Component | power calendar multiple', function(hooks) {
       callsCount++;
       if (callsCount === 1) {
         assert.equal(days.date.length, 1);
-        assert.ok(isSame(days.date[0], new Date('2013-10-05'), 'day'));
```

Comment:
Not that I oppose, by why changing this? Aren't those equivalent?

Reply:
In chrome `new Date('2013-10-05')` creates a date at 2013-10-05 UTC.  but the project expects 2013-10-05 LOCAL time.  `new Date(2013, 9, 5)` gives a local time date.